docs/gosec.md

```diff
@@ -0,0 +1,42 @@
+# GOSEC static analysis 
+## Background
+Gosec (https://github.com/securego/gosec) is a static code analysis tool that scans the code to find security related issues.
+
+Gosec executes a set of validation tests/rules and produces a report listing all the detected potential violations. The set of rules can be configured as well as include and exclude directories. 
+
+## Running gosec in your local environment
+To perform gosec scanning over your code run:
+```
+    make gosec
+```
+This command will execute all the tests. If you want to run a specific gosec test you can use the `GOSEC` environment variable: E.g.,  `GOSEC="G601`"
+
+## Handle false positive detected issues
+As a static analysis tool, gosec may produce false positive warnings - identify a potential risk that is not relevant/applicable in the specific KubeVirt code. 
+In extreme cases you can annotate the code with a special comment:
+```
+// #nosec <description>
+```
+This will instruct gosec to suppress the warning.
+The developer is encourage to avoid such annotation when possible as this will silence the warnnings. 
```

Comment:
I'd stress that it would silence *any* warning in the line below the comment, even if the code changes and another, true positive issue is created.

Reply:
Done.